app/views/users/show.html.erb

```diff
@@ -3,3 +3,18 @@
 </div>
 
 <h1>Hello, <%= current_user.email %>!</h1>
+
+<h2>Viewing Parties</h2>
+
```

Comment:
When we create our "parties" model the controller can create @hosted_parties and @invited_parties. I am excited to work on this!